learn/ai_powered_search/getting_started_with_ai_search.mdx

```diff
@@ -17,72 +17,154 @@ This tutorial will walk you through configuring AI-powered search in your Meilis
+  "kitchenware-openai": {
+    "source": "openai"
+  }
+}
+```
+
+Meilisearch supports several embedder sources. OpenAI is a good option for most use cases.
+
+### Choose an embedder model
+
+Models supply the information required for embedders to process your documents.
+
+Add a new `model` field to your embedder object:
+
+```json
+{
+  "kitchenware-openai": {
+    "source": "openai",
+    "model": "text-embedding-3-small"
+  }
+}
+```
 
-Next, you must generate vector embeddings for all documents in your dataset. Embeddings are mathematical representations of the meanings of words and sentences in your documents. Meilisearch relies on external providers to generate these embeddings. This tutorial uses an OpenAI embedder, but Meilisearch also supports embedders from HuggingFace, Ollama, and any embedder accessible via a RESTful API.
+Models are usually created with specific use cases in mind. `text-embedding-3-small` is a cost-effective model for general usage.
 
-Use the `embedders` index setting of the [update `/settings` endpoint](/reference/api/settings?utm_campaign=vector-search&utm_source=docs&utm_medium=vector-search-guide) to configure an [OpenAI](https://platform.openai.com/) embedder:
+### Create your API key
+
+Log into OpenAI, or create an account if this is your first time using it. Generate a new API key using [OpenAI's web interface](https://platform.openai.com/api-keys).
+
+Add the `apiKey` field to your embedder:
+
+```json
+{
+  "kitchenware-openai": {
+    "source": "openai",
+    "model": "text-embedding-3-small",
+    "apiKey": "OPEN_AI_API_KEY",
+  }
+}
+```
+
+Replace `OPEN_AI_API_KEY` with your own API key.
+
+<Capsule intent="tip" title="OpenAI key tiers">
+You may use any key tier for this tutorial. Use at least [Tier 2 keys](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-two) in production environments.
+</Capsule>
+
+### Design a prompt template
+
+Meilisearch embedders only accept textual input, but documents can be complex objects containing different types of data. This means you must convert your documents into a single text field. Meilisearch uses [Liquid](https://shopify.github.io/liquid/basics/introduction/), an open-source templating language to help you do that.
+
+A good template should be short and only include the most important information about a document. Add the following `documentTemplate` to your embedder:
+
+```json
+{
+  "kitchenware-openai": {
+    "source": "openai",
+    "model": "text-embedding-3-small",
+    "apiKey": "OPEN_AI_API_KEY",
+    "documentTemplate": "An object used in a kitchen named '{{doc.name}}'"
+  }
+}
+```
+
+This template starts by giving the general context of the document: `An object used in a kitchen`. Then you give it the information specific to each document. `doc` is your document, and you can access any of its attributes using dot notation. `name` is an attribute in your document, with values such as `wooden spoon` or `rolling pin`.
+
```

Comment:
I think we should specify here that if a field appears in a document template, then it should exist in all the documents of the index?